ui/webui/test/README.rst

```diff
@@ -13,10 +13,14 @@ Preparation and general invocation
 To run the WebUI integration tests run the following from the root of the anaconda repo.
 (do NOT run the integration tests as root)::
 
-You first need to build anaconda RPMS::
+You first need to build anaconda RPMS in the repository root::
 
+    ./autogen.sh
```

Comment:
Instead I would point to https://github.com/rhinstaller/anaconda/blob/master/CONTRIBUTING.rst#how-to-run-make-commands